sql/server/core/src/main/kotlin/core/Skdb.kt

```diff
@@ -22,9 +22,16 @@ data class ProcessOutput(val output: ByteArray, val exitCode: Int) {
     return exitCode == 0
   }
 
-  fun getOrThrow(): ByteArray {
+  fun logOnError(): ProcessOutput {
     if (exitCode != 0) {
       System.err.println(decode())
+    }
+    return this
+  }
+
+  fun getOrThrow(): ByteArray {
+    if (exitCode != 0) {
```

Comment:
nit: some code tests `exitCode` and some code calls `exitSuccessfully`, is there a pattern?

Reply:
exitSuccessfully is the interface. The implementation was just using the field (which admittedly isn't private so could be used externally). I change the implementation to call exitSuccessfully.